lms/djangoapps/student_dashboard/tests.py

```diff
@@ -0,0 +1,36 @@
+import ddt
+import mock
+import factory
+
+from django.db.models import signals
+from django.test import TestCase, RequestFactory
+
+from common.lib.nodebb_client.categories import ForumCategory
+from lms.djangoapps.onboarding.tests.factories import UserFactory
+from lms.djangoapps.student_dashboard.views import get_joined_communities, get_recommended_xmodule_courses, \
+    get_enrolled_past_courses
+
+
+@ddt.ddt
+class TestStudenDashboard(TestCase):
+
+    @factory.django.mute_signals(signals.pre_save, signals.post_save)
```

Comment:
This is a great idea! Can we it at `UserFactory` level? There a tests which are failing because of the hits nodebb, mail_chimp and mandrill the user post save makes. If that is possible those tests will be automatically fixed.

Reply:
🤞 not sure if we can do this with `pytests`

Reply:
it will also work with pytests

Reply:
understood it will work with pytest. But I am asking can we mute these signals inside UserFactory? It will benefit us a lot, many of our existing failing unit tests will be fixed automatically.

Reply:
@ahmed-faraz46 apparently we can.

Reply:
Can we try in the scope of this ticket?